pages/linux/slop.md

```diff
@@ -0,0 +1,24 @@
+# slop
+
+> Get a selection of the screen.
+> More information: <https://github.com/naelstrof/slop>.
+
+- Wait for the user to make a selection and output its geometry to standard output:
+
+`slop`
+
+- Double click, rather than click and drag, to draw a selection:
+
+`slop -D`
+
+- Highlight, rather than outline, the selection:
```

Comment:
Perhaps this will read better?
```suggestion
- Highlight rather than outline the selection:
```

Reply:
Resolved in 84aec1cb576da76a27f2d9947708b7ceb95416d2